lounge/docker-compose.yml

```diff
@@ -0,0 +1,8 @@
+lounge:
+  image: thelounge/lounge:latest
```

Comment:
It pulls whatever version is defined in the `Dockerfile` from GitHub right now. This also means it packs lounge's `devDependencies` in the image in order to be able to build the app (somewhere in the range of 80M increase iirc, just for the `postinstall` script). So the current image is very bloated and has work to be done in terms of reducing its size. I'll move to pulling the package from `npm` in the future - a simple fix.

<sup>(btw, my previous, bad, `NODE_ENV=production` commit broke the `postinstall` script since it implies `npm install --production`)</sup>

I'll have to read up on how to manage automated dockerhub deployments as well.